slider/beatmap.py

```diff
@@ -23,6 +24,9 @@
 from .curve import Curve
 
 
+LAZY_SLIDER_END_OFFSET = 36
```

Comment:
why is this called `lazy`? maybe a typo of `legacy`?

Reply:
It is indeed a bit confusing. Its called lazy here because it causes the slider end to come earlier than the actual end is.

The cause is some legacy hack initially added by peppy, but the resulting behavior is more akin to lazy in a gameplay sense. Rename is ofc possible, i named it in accordance to the gameplay aspect, as that's the only way to encounter it.

Reply:
I'm not sure I see how the slider end coming earlier is related to the slider end (or really anything) being lazy.

I've changed it to `LEGACY_LAST_TICK_OFFSET` in line with lazer.